src/concrete/ml/sklearn/xgb.py

```diff
@@ -331,6 +331,14 @@ def fit(self, X, y, *args, **kwargs) -> Any:
         super().fit(X, y, *args, **kwargs)
         return self
 
+    def post_processing(self, y_preds: numpy.ndarray) -> numpy.ndarray:
+        y_preds = super().post_processing(y_preds)
+
+        # Hummingbird Gemm for XGBoostRegressor adds a + 0.5 at the end of the graph.
+        # We need to add it back here since the graph is cut before this add node.
+        y_preds += 0.5
```

Comment:
That's what I was looking for in https://github.com/zama-ai/concrete-ml/pull/355/files lol

thanks